WordPress/src/main/java/org/wordpress/android/ui/main/SitePickerAdapter.java

```diff
@@ -433,7 +436,14 @@ void setEnableEditMode(boolean enable) {
         }
 
         mIsMultiSelectEnabled = enable;
-        mSelectedPositions.clear();
+
+        // If some selectedPositions exist and are passed from the activity, use it.
+        // Otherwise, reset the selected positions.
```

Comment:
Just some suggestions to improve the comment a bit. Let me know what you think @hafizrahman 😄 
```suggestion
        // If the selectedPositions value that is passed from the activity isn't empty then we add that to the Adapter's mSelectedPositions otherwise we reset the selected positions .
        // Otherwise, reset the selected positions.
```

Reply:
Yup! Sounds so much better. Added in the next commit.